derive/Cargo.toml

```diff
@@ -16,7 +16,7 @@ panic = "abort"
 
 [dependencies]
 quote = "0.6.8"
-syn = { version = "0.15", features = ["full", "extra-traits"] }
+syn = { version = "0.15.12", features = ["full", "extra-traits"] }
```

Comment:
any reason to be specific with patch version?

Reply:
This PR uses `syn::AttributeArgs` which have been introduced in `15.10`.
I used `0.15.12` since it is the most recent version. Also `15.11` looks like a fix version for things that are connected to `syn::AttributeArgs`.